### PR TITLE
Review path planning n

### PR DESCRIPTION
**Pull request by NiklasDahlquist1**

### CODE REVIEW

File: report/finished/chapters/motion.tex [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+
+\subsection*{Joint Space interpolation or Cartesian Motion}
+The interpolation can take place in joint space or in cartesian space. Joint space interpolation means that the waypoints are expressed as joint values and that the interpolation is calculated between these joint values. Cartesian motion means that the waypoint are expressed as cartesian coordinates and that the interpolation takes place in cartesian space.
+
+%Joint space - interpolate the joint values
+%Cartesian motion - interpolate in 3D space -> convert to joint angles
+\subsection*{Implementation} %Pre recorded path
+The robotic manipulator, shown in figure \ref{fig:robotic_manipulator_general}, has a number of waypoints that it is required to visit. The desired waypoints are expressed in joint values, and a cubic spline interpolation is used to interpolate between these joint space waypoints. The path for picking up and placing the object can be seen in figure \ref{fig:pickup_path_interpolated} and \ref{fig:place_path_interpolated} respectively.
```
Comment (edwced) [inline report/finished/chapters/motion.tex:17]: "that it is required to visit"
dont know if this sentence is needed.

File: report/finished/chapters/motion.tex [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+%Path planning is very important, a smooth path will make a manipulator move better :) %is right
+\subsection*{Interpolation}
+One method of interpolating between points is to use cubic spline interpolation. A spline \cite{spline_wolfram} is a piecewise polynomial function and consequently a cubic spline is a function consisting of piecewise cubic polynomials. A cubic polynomial will ensure a smooth movement since its first derivative, the velocity, will be continuous. To ensure that each required point is visited the bounding values for each cubic function are set to zero \cite{cubic_spline_wolfram}.
+
+\subsection*{Joint Space interpolation or Cartesian Motion}
+The interpolation can take place in joint space or in cartesian space. Joint space interpolation means that the waypoints are expressed as joint values and that the interpolation is calculated between these joint values. Cartesian motion means that the waypoint are expressed as cartesian coordinates and that the interpolation takes place in cartesian space.
```
Comment (Partins) [inline report/finished/chapters/motion.tex]: mention that joints are angles

File: report/finished/chapters/motion.tex (hunk 1 above)
Comment (Partins) [inline report/finished/chapters/motion.tex:17]: When refering to figures use "... Fig. \ref{}

File: report/finished/chapters/motion.tex [hunk 3]
```diff
@@ -1,4 +1,40 @@
-\section*{Robot arm}
+\section*{Robot Arm Path Planning} 
+
+
+
+If a robot manipulator is required to visit specific discrete points, waypoints, path planning is the problem of deciding how to move between these points. Planning the path is also important to ensure the movements are smooth, meaning that, since we "control" the speed of each joint, the desired positions and velocities should be continuous.
```
Comment (Partins) [inline report/finished/chapters/motion.tex:5]: We don't "control", we actually control, even if it's open loop so you can remove the citations

**Review by Partins — COMMENTED** [review]
Good overall, very nice that there's a explanation of cubic polynomial, what cartesian space is etc. Just make sure to refer to figures as "... Fig . \ref{}"

File: report/finished/chapters/motion.tex (hunk 2 above)
Comment (NiklasDahlquist1) [reply report/finished/chapters/motion.tex]: Good point, joint values can be either angles or distance. I will add it

**Pull request merged**